### DISCUSSION TITLE
Using a custom domain name with Pages and can't access Webmail

### DISCUSSION THREAD

**Discussion by noahbeckwith** [community/community#22725] [Other Feature Feedback, Questions, & Ideas]
<p>I have my <a href="http://github.io" rel="nofollow noopener">github.io</a> configured with a custom domain name, but I get a 404 page when I try to access my webmail account at mail.(mydomainname).com</p>
<p>How can I fix this? I basically just want to log in there once so I can get a code to set the email up in Gmail.</p>

**Comment by otter-computer** [comment]
<p>This could be an issue with the DNS setup for your domain, or could be an issue with the type of site you’ve hosted on GitHub Pages. Without being able to see the site and the domain I can’t be sure exactly.</p>
<ul>
<li>Would you be able to post a link to your site?</li>
<li>Is the site you’re hosting a single page web app?</li>
<li>When setting up your domain, do you remember exactly which DNS settings you changed/added to point it towards GitHub Pages?</li>
</ul>

**Reply by Savannaha83** [reply]
The domain had been linked with Norton lock for life and to my Acer desktop along with kernel rooted with my What's app

**Comment by noahbeckwith** [comment, marked as answer]
<p>I actually was just able to solve it. The issue was with the MX record.</p>
<p>Thanks for the info!</p>

**Reply by Savannaha83** [reply]
Alright can information about MX records be forwarded to me

**Comment by Savannaha83** [comment]
If all information linked can be forwarded to me in the provide d emails
